tools/setup.sh

```diff
@@ -0,0 +1,12 @@
+VERSION=0.6.1
+ARCH=$(uname -m)
+
+echo "=> Downloading HEMTT..."
+wget -q --show-progress https://github.com/synixebrett/HEMTT/releases/download/v$VERSION/hemtt-v$VERSION-$ARCH-unknown-linux-gnu.tar.gz
```

Comment:
The script will continue even if this fails. Add `set -e` somewhere before to exit the script if any command fails.